src/common/const.ts

```diff
@@ -77,6 +77,7 @@ export const DOMAINS_TOGGLE = new Set([
   "input_boolean",
   "light",
   "switch",
+  "group",
```

Comment:
We shouldn't do that, we should consider using `canToggleDomain` and `canToggleState`

Reply:
I think I’m missing something, what would they do?

Reply:
Oh wait, is this just for the services being called?

Reply:
It seems to be used in 3 places: to check if a `group-entity-row` should show display a toggle (was a bug in that), to see what entities should be toggled in `hui-entities-toggle` and what should be done in `picture-glance`, toggle or dialog. The last one might be a problem...